pkgs/build-support/node/fetch-npm-deps/src/main.rs

```diff
@@ -182,6 +182,15 @@ fn main() -> anyhow::Result<()> {
         process::exit(1);
     }
 
+    if let Ok(jobs) = env::var("NUM_JOBS") {
```

Comment:
Shouldn't be `NIX_BUILD_CORES`?

Reply:
I thought stdenv set both, and figured this was a more general-use variable (I think `buildRustPackage` stuff uses `NUM_JOBS` too, but I'd have to double check when I'm not on mobile). I'm happy to change it though since I don't feel particularly strongly on it

Reply:
Done, thanks for catching that!